sdk/applicationinsights/ci.yml

```diff
@@ -13,18 +27,19 @@ pr:
   branches:
     include:
     - master
+    - feature/*
+    - hotfix/*
+    - release/*
   paths:
     include:
     - sdk/applicationinsights/
 
-resources:
-  repositories:
-    - repository: azure-sdk-tools
-      type: github
-      name: Azure/azure-sdk-tools
-      endpoint: azure
-
-jobs:
-- template: ../../eng/pipelines/templates/jobs/archetype-sdk-client.yml
+stages:
+- template: ../../eng/pipelines/templates/stages/archetype-sdk-client.yml
   parameters:
-    ServiceDirectory: applicationinsights
\ No newline at end of file
+    ServiceDirectory: applicationinsights
+    ArtifactName: packages
+    Artifacts:
+    - name: Microsoft.Azure.ApplicationInsights
```

Comment:
In this case the package name is what we need because I use the filtering logic to pick-up the nupkg files from the artifacts. I'll fix up the ci.yml file.